src/main/resources/templates/staySafe.html

```diff
@@ -4,17 +4,13 @@
 <!-- Required meta tags -->
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="styles.css"> 
 
-<!-- Required meta tags -->
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-
-    
 <title>Stay Safe</title>
 </head>
 <body>
 	<div class="container">
-		<h4>Patient Information</h4>
```

Comment:
Mozilla html styleguide, "Avoid using heading elements to resize text. Instead, use the CSS font-size property. Avoid skipping heading levels: always start from <h1>, followed by <h2> and so on."

I changed the h4 tag to an h1 tag since the usage notes recommend starting at h1. I achieved the same styling as an h4 tag by linking a css stylesheet to the html page that resizes the h1 tag to 1em.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements